source/connection-monitoring-and-pooling/connection-monitoring-and-pooling.rst

```diff
@@ -523,6 +523,16 @@ occur only while the pool is "ready". If the pool implements a background
 thread, it can be used for this. If the pool does not implement a background
 thread, the checkOut method is responsible for ensuring this requirement is met.
 
+When populating the Pool, pendingConnectionCount has to be decremented after
+establishing a `Connection`_ similarly to how it is done in
```

Comment:
In my mind "after establishing a `Connection`_" does not convey the nuance required here. I think we need to say that pendingConnectionCount must not be decremented until after the established Connection is actually added to the pool.

Reply:
The first sentence is not there to specify a requirement (even though it must be so), but to introduce the notion of a specific signal, for which an ordering constraint is then specified. I updated the ordering constraint such that it refers to the "Marking a Connection as Available" action, which is the wording the specification uses to say "add connection to the pool".

Reply:
Ah thanks I get it now.